src/BlobBuffer.js

```diff
@@ -220,7 +220,7 @@
 	};
 	
 	if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {
-		module.exports = BlobBuffer(require('fs'));
+		module.exports = BlobBuffer;
 	} else {
 		window.BlobBuffer = BlobBuffer(null);
```

Comment:
It feels a little odd to me to have one platform exporting a function and the other exporting the return of the function. Maybe change `main.js` to call the function like `browser.js` does?

As a future features, I've also [seen a pattern in other libraries to pass in fs as an external](https://isomorphic-git.org/docs/en/fs). It'd be nice if the user could just pass in whatever implementation of `fs` for whatever their needs are, meaning you could use anything from [`BrowserFS`](https://github.com/jvilk/BrowserFS)